LandSandBoat/server
# [SQL] Remove SDT values and add resistance ranks to first 63 entries in mob_resistances

**_I affirm:_**

## What does this pull request do?
- Removes values from SDT columns when applicable
- Adds values to resistance rank columns.

Decided to go in small chunks couse this stuff bores my brains out.
Data from this chart:
https://docs.google.com/spreadsheets/d/1YBoveP-weMdidrirY-vPDzHyxbEI2ryECINlfCnFkLI/edit#gid=1367309866

Data entry from Spicy, doublechecked by me.

## Steps to test these changes

Import the table properly. See affected mobs get more (or less) resists in both nukes and status effects (spells) and be able in some cases to be immunobroken even.
